### ISSUE TITLE
Remove `sddi-social` image variant

### ISSUE THREAD

**Issue by BWibo** [tum-gis/ckan-docker#60]
This has never been completed and has never been used. Do we really need it @MarijaKnezevic?

**Comment by thomashkolbe** [comment]
It might become useful again in the future. Does the inclusion of this image cause signficant extra work? If yes, we could also think about hibernating the image and to add an information at appropriate places that there is also the possibility to add public commenting of each dataset using Disqus (with a pointer to the repo), but that this facility is currently not maintained.
@BWibo, @MarijaKnezevic: What do you think?

**Comment by MarijaKnezevic** [comment]
@BWibo yes true. This function was never used. We can document the idea for this in the "Future Ideas Features" PR, but for now, to reduce maintenance, this can be deleted or mentioned that this image is only going to be updated if this funcionality will be required.

**Comment by BWibo** [comment]
It's not really causing any extra work, but the extension we use does not seem to be maintained (last commit 2 years ago).
And we never tested if this is atually working.

**Comment by BWibo** [comment]
Resolved by #71.
